README.md

```diff
@@ -1,11 +1,53 @@
-# *A*ndroid *L*ibrary for *P*assport *I*nformation via *N*FC *E*xtraction
+# **A**ndroid **L**ibrary for **P**assport **I**nformation via **N**FC **E**xtraction
 
-## Usage
+
+ALPINE is a library to simplify the extraction of passport information by reading data off of the NFC chip
+
```

Comment:
One more thing, I think it would be great If we leave some screenshot example below here 🤔